ctapipe/image/muon/muon_diagnostic_plots.py

```diff
@@ -139,19 +141,20 @@ def plot_muon_event(event, muonparams):
                 array_direction=altaz,
                 pointing_direction=altaz)
 
+            flen = subarray.tel[tel_id].optics.equivalent_focal_length
             ring_camcoord = ring_nominal.transform_to(CameraFrame(
                 pointing_direction=altaz,
-                focal_length=event.inst.optical_foclen[tel_id],
+                focal_length=flen,
                 rotation=rotr_angle))
 
             centroid = (ring_camcoord.x.value, ring_camcoord.y.value)
 
             ringrad_camcoord = muonparams['MuonRingParams'][idx].ring_radius.to(
                 u.rad) \
-                * event.inst.optical_foclen[tel_id] * 2.  # But not FC?
+                * flen * 2.  # But not FC?
```

Comment:
This comment contains a question mark. To me this looks, like the author was unsure if this is right. This might be a sign for a missing test, to clarify this matter.

Reply:
But the comment was here before this PR and it was not touched by the PR. So this PR is not responsible to tidy up this comment. But I thought maybe the author of this PR knows if the comment makes sense or not.

I do not understand the comment at all.

Reply:
Can this test by hand be converted in a unit test? E.g. Testing if most of the pixels are close to the ring?

Reply:
Something on the line of this, although a bit different, was already proposed in #615 by @watsonjj. I can open an Issue for a test to be created.